packages/widget/src/components/network-selector/network-selector.ts

```diff
@@ -16,70 +17,91 @@ type Direction = (typeof Directions)[keyof typeof Directions];
 export class NetworkSelector extends LitElement {
   static styles = styles;
 
-  @property({
-    type: Boolean
-  })
-  disabled = false;
+  @state()
+  _isDropdownOpen = false;
 
-  @property({
-    type: Boolean
-  })
-  icons = true;
+  @property({ type: Boolean })
+  disabled = false;
 
-  @property({
-    type: String
-  })
+  @property({ type: String })
   direction?: Direction;
```

Comment:
what if default value of direction is from? wdty?

Reply:
I think it's better to specify explicitly in such case or render no label. I don't have a strong opinion here